### PR TITLE
add "create with explicit ID" tests on field, fieldset, item, itemType, itemTypeFilters, menuItem, plugin, upload, uploadFilters and workflow

### PR DESCRIPTION
**Pull request by pinzer**

### CODE REVIEW

File: packages/cma-client-node/__tests__/item.test.ts [hunk 1]
```diff
@@ -276,4 +277,33 @@ describe('item', () => {
...
+
+    const item = await client.items.create({
+      id: newId,
+      title: 'My first blog post',
+      item_type: itemType,
+    });
+
+    await client.items.bulkPublish({
+      items: [item],
+    });
```
Comment (stefanoverna) [inline packages/cma-client-node/__tests__/item.test.ts]: The client.items.bulkPublish here is not useful, is not part of what we want to test.

**Review by stefanoverna — CHANGES_REQUESTED** [review]

File: packages/cma-client-node/__tests__/item.test.ts (hunk 1 above)
Comment (pinzer) [reply packages/cma-client-node/__tests__/item.test.ts]: correct, i will remove it, thanks

**Review by sistrall — COMMENTED** [review]
Tests don't run on CI because – I suppose – API PR about this feature is not released, yet. Am I correct?

**Comment by pinzer** [comment]
> Tests don't run on CI because – I suppose – API PR about this feature is not released, yet. Am I correct?

correct, works on changes of the schema that are not yet in production

**Pull request merged**